jobs/config.json

```diff
@@ -3259,6 +3283,29 @@
       "sig-scalability"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-kube-dns": {
+    "args": [
+      "--cluster=gce-scale-cluster",
+      "--env=HEAPSTER_MACHINE_TYPE=n1-standard-8",
```

Comment:
if you want this test for "kube-dns" whatever the default DNS server, you may want to add the flag:
"--env=CLUSTER_DNS_CORE_DNS=false"